buku

```diff
@@ -3375,6 +3383,28 @@ def convert_bookmark_set(
             count += 1
 
         out += '</xbel>'
+    elif export_type == 'rss':
+        timestamp = datetime.datetime.now().isoformat()
+        out = (
+            '<feed xmlns="http://www.w3.org/2005/Atom">\n'
+            '<title>Bookmarks</title>\n'
+            '<generator uri="https://github.com/jarun/buku">buku</generator>\n'
+            '<updated>%s</updated>\n' % timestamp
+        )
+
+        for row in resultset:
+            out += '<entry>\n'
+            out += '        <title>' + title(row) + '</title>\n'
+            out += '<link href="%s" rel="alternate" type="text/html"/>' % html.escape(row.url).encode('ascii', 'xmlcharrefreplace').decode('utf-8')
+            out += '        <id>%s</id>\n' % row.id
+            for tag in filter(lambda t: len(t) > 0, row.tags.split(',')):
```

Comment:
This would work just as well:
````py
for tag in (t for t in row.tags.split(',') if t):
````
…Also, [the predicate passed to `filter()` defaults to identity](https://docs.python.org/3/library/functions.html#filter): `filter(None, row.tags.split(','))`

Reply:
Fixed!